index.html

```diff
@@ -4,95 +4,99 @@
-				<p class="little-intro">Welcome to my world of</p>		
-				<p class="little-intro">organised chaos!</p>		
-			 </section>
+	<body>
+		<header class="id-section-bar">
+			<a class="id-section-bar" href="#about">About Me</a>  
+			<a class="id-section-bar" href="#portfolio">Portfolio</a>
+			<a class="id-section-bar" href="#contact">Contact information</a>		
+		</header>
+		<section class="intro">
+			<p class="intro">HEELLL<br>OOOOO</p>
+			<h1 class="second-intro">I am Josh</h1>
+			<p class="second-intro">Welcome to my world of</p>
+			<p class="second-intro">organised chaos!</p>		
+		</section>
 			 
-	 		<div id="about" class="about-catcher">
-				<section class="about">
-	 				<h2 class="about">About Me</h2>
-	 					<p class="about"> Alright Mate. I am Josh. I have lived a life of organised chaos and have now found myself in the depths of coding.</p>
-						<p class="about"> I started out as an Artist beleive it or not, focused on both the buissness and creative side of the art world. My biggest passion was the 					digital arts and creating stories from a single image.</p>
-						<p class="about"> After some time in the art world and the Hospitality world, I have moved onto the path of coding.</p>
-						<p class="about"> A new way to be creative and to tell a story.</p>
-						<p class="about">A new way of organised chaos!</p>
-	 				<img class="about-img" src="joshpic.jpg" alt="A Picture of Josh">
-	 			</section>
+		<section class="about-section" id="about"> 
+			<div class="about-piece">
+			<div class="about-contents">
+	 			<h2 class="about-contents">About Me<br></h2>
+	 			<p class="about-contents"> Alright Mate. I am Josh. I have lived a life of organised chaos and have now found myself in the depths of coding.</p>
+				<p class="about-contents"> I started out as an Artist beleive it or not, focused on both the business and creative side of the art world. My biggest passion was the digital arts and creating stories from a single image.</p>
+				<p class="about-contents"> After some time in the art world and the hospitality world, I have moved onto the path of coding.</p>
+				<p class="about-contents"> A new way to be creative and to tell a story.</p>
+				<p class="about-contents">A new way of organised chaos!</p>
+	 			<img class="about-image" src="joshpic.jpg" alt="Josh">
 			</div>
+			</div>
+		</section> 
 			
-			<div id="portfolio" class="portfolio-catcher">
-				<section class="portfolio-intro">
-					<h2 class="portfolio-intro"> Portfolio</h2>
-					<p class="portfolio-intro"> Here is a show case of my work so you can see how efficient and proficient yet chaoticly organised I am.</p>
-				</section>
-				</div>
+			<div id="portfolio">
+			<section class="portfolio-intro">
+				<h2 class="portfolio-intro"> Portfolio</h2>
+				<p class="portfolio-intro"> Here is a show case of my work so you can see how efficient and proficient yet chaoticly organised I am.</p>
+			</section>
+			</div>
```

Comment:
1. in general, need to make sure you are not using the same class name for parent and child
2. here you have a div with only 1 child which is a section - you probably don't need both and can get away with having just one of them